### DISCUSSION TITLE
Request Transformer - Unable to remove querystring(groupby)

### DISCUSSION THREAD

**Discussion by sivasubramani300890** [Kong/kong#9559] [Help]
Hi team,

I have tried removing query string using request_transformer

![image](https://user-images.githubusercontent.com/81576773/195982330-cfa704b7-8440-4539-9ea1-e01f334ecd8b.png)

But it didn't helped to get right response

**Comment by StarlightIbuki** [comment]
Hi, thank you for reporting this. Could you provide more details, like what response is expected and what you got? And you could check the upstream log to see if the query argument is stripped.

**Comment by sivasubramani300890** [comment]
Yes, its working now,Thanks!

**Comment by sivasubramani300890** [comment]
Is it possible to alter query string value via request transformer plugin?
